tests/benchmark_test.py

```diff
@@ -131,3 +133,131 @@ def test_start_project_from_scratch(mock_collect_user_input):
+        " standalone. Each one needs to be included. Note that the father is listed"
+        " first in a parents list."
+    )
+    # Removes the line: "Make sure to pay special attention to the nuances of loading the data."
+    # 15 passed, 5 failed - 75% pass rate
+    easy_medium_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " In this data file people are stored either as parents, children, or"
+        " standalone. Each one needs to be included. Note that the father is listed"
+        " first in a parents list."
+    )
+    # Removes the line: "Each one needs to be included."
+    # 8 passed, 12 failed - 40% pass rate
+    medium_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " In this data file people are stored either as parents, children, or"
+        " standalone. Note that the father is listed first in a parents list."
+    )
+    # Removes the Line: "In this data file people are stored either as parents, children, or standalone."
+    # Probably 0% pass rate (I ran it a number of times)
+    hard_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " Note that the father is listed"
+        " first in a parents list."
+    )
+    # makes linter shut up
+    easy_prompt
+    easy_medium_prompt
+    medium_prompt
+    hard_prompt
+    mock_collect_user_input.side_effect = [
+        easy_medium_prompt,
+        "y",
+        KeyboardInterrupt,
+    ]
+    run(["person_data/data.json", "person_data/person.py"])
+
+    # This imports the Person class from the file mentat edited just created
+    # import in Python is always relative to the file's original path
+    # so even though we change the cwd, we can't use relative imports to import form the tmp dir
+    file_path = os.path.join(os.getcwd(), "person_data/person.py")
```

Comment:
I can see us wanting to import things in benchmarks a lot; could be good to make this a helper function in some test_utils file or something. Not necessary in this PR though if you don't want to.

Reply:
In this PR lets put this import stuff in a helper function. We can create a `test_utils` in a future PR when we need to use it somewhere else as well (and better understand what we want to abstract).

Reply:
Done